requirements.txt

```diff
@@ -8,3 +8,5 @@ Werkzeug>=3.0.3,<4
 schemathesis~=3.7
 hypothesis~=6.13
 lxml-stubs~=0.5.1
+rdflib~=7.0.0
```

Comment:
We deprecated the `requirements.txt`, so please put these into the `pyproject.toml`. You may want to merge the current state of `eclipse-basyx/main` into your branch in order to get the working state of the `pyproject.toml`. 

After moving your dependencies, you can delete the `requirements.txt`, as it's not needed anymore.

Reply:
Ok, should be hopefully be fine now in my latest commits